src/types/Filter.ts

```diff
@@ -0,0 +1,5 @@
+export enum Filter {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
```

Comment:
It's better to capitalize the Enum properties
https://www.typescriptlang.org/docs/handbook/enums.html